Car_Collision_Visulaization/main.cpp

```diff
@@ -1,27 +1,30 @@
-#include <chrono>  
+#include <chrono>
 #include <iostream>
-#include <thread>  
+#include <thread>
 
 #include "src/car.h"
 #include "src/visualizer.h"
 
 int main()
-{
-  std::vector<Car> all_cars;
-  all_cars.push_back(Car(200, 2, 5, 45, 30, 10));
-  all_cars.push_back(Car(2, 2, 5, 45, 30, 10));
-
-  cimg::wait(100);
+ {
+  cimg_library :: cimg :: wait(100);
   Visualizer visualizer(800, 800);
 
-  while (true)
+  std::vector<Car> all_cars;
+  all_cars.push_back(Car(visualizer.length(), visualizer.width()/2, 5, 180, 30, 10));
+  all_cars.push_back(Car(0                  , visualizer.width()/2, 5, 0  , 30, 10));
```

Comment:
This should not be dependent on visualizer length and width. They should be separate